docs/content/docs/flinkDev/ide_setup.md

```diff
@@ -228,6 +228,12 @@ the Project SDK.
 
 When switching back to newer Flink versions you may have to revert this change again.
 
+#### Compilation fails with `package sun.misc does not exist`  
+
+This happens if you are using JDK 11 as project SDK and compiling with `--release` option target 8 when running test in IDE.
+Which can be solved by disable the `--release` compiler option [JEP-247](https://openjdk.java.net/jeps/247). Go to 
+"Preferences" → "Build, Execution, Deployment" → "Compiler" → "Java Compiler" and uncheck the "Use '--release' option for cross-compilation (Java 9 and later)".
```

Comment:
```suggestion
This happens if you are using JDK 11 and compile to Java 8 with the `--release` option. This option is currently incompatible with our build setup.
Go to "Settings" → "Build, Execution, Deployment" → "Compiler" → "Java Compiler" and uncheck the "Use '--release' option for cross-compilation (Java 9 and later)".
```

I don't think the JEP link is that useful, and it would be nice to also explain that this an issue with Flink itself.

Reply:
ok，I will change the chinese doc accordingly.